### DISCUSSION TITLE
what is the ECCN code of clickhouse?

### DISCUSSION THREAD

**Discussion by chimw** [ClickHouse/ClickHouse#49248] [Q&A] [answered]
what is the ECCN code of clickhouse?

**Comment by thomoco** [comment, marked as answer]
Hi @chimw - we discussed this in a Support Case also recently, but replying here also for public visibility.

Open-source software products are not assigned ECCNs because they are "published" (i.e., the source code is available) and are therefore outside the scope of Export Administration Regulations (EAR) jurisdiction. Our open-source software does not have an ECCN because it is not applicable.
